snow-day/convert_array.py

```diff
@@ -0,0 +1,11 @@
+"""Convert a number (n) in a reverse list of the individual numbers."""
+
+
```

Comment:
Good work getting this one working, but let's try to optimize this a little bit.  We know we are building a list, so a comprehension here would probably work.  We can also easily reverse a list also using [::-1] syntax.  If we use those bits of info, you should eventually be able to do this in one line like:
return [int(x) for x in list(str(a))[::-1]]
By the end of this class, you will be able to do this like a boss!